### PR TITLE
Map upload refactor + validation

### PR DESCRIPTION
**Pull request by Brutus5000**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -204,6 +204,14 @@ dependencyManagement {
   }
 }
 
+test {
+  useJUnitPlatform {
+  }
+  testLogging {
+    events("passed", "skipped", "failed")
```
Comment (1-alex98) [inline build.gradle:211]: Would that also be interesting for the client?

Comment (Brutus5000) [reply build.gradle:211]: Probably.

File: src/main/resources/templates/validate_map_metadata.html [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+                      $("#mapScenarioTable").append('<tr class="result"><td>Result</td><td>No errors</td>');
+                  })
+                  .fail(function (xhr, status, error) {
+                      xhr.responseJSON.errors.forEach(function (nextError) {
+                          $("#mapScenarioTable").append('<tr class="result"><td>Error</td><td>' + nextError.detail + '</td>');
+                      });
+                  });
+          });
+
+          /*$('#validateMapScenario').click(function() {
```
Comment (1-alex98) [inline src/main/resources/templates/validate_map_metadata.html]: Remove?

**Review by 1-alex98 — DISMISSED** [review]

File: src/main/resources/templates/validate_map_metadata.html [hunk 3]
```diff
@@ -4,105 +4,10 @@
...
-      display: inline-block;
-      opacity: 0.6;
-      transition: opacity ease 0.5s;
-    }
-
-    .alert {
-      color: darkred;
-    }
-  </style>
+  <link href="https://fonts.googleapis.com/css?family=Yanone+Kaffeesatz" rel="stylesheet" type="text/css"/>
```
Comment (1-alex98) [inline src/main/resources/templates/validate_map_metadata.html:7]: Kaffesatz Xd

**Review by 1-alex98 — DISMISSED** [review]

**Comment by Brutus5000** [comment]
@axel1200 You should really test out the stuff you approve. Then you would have noticed that the css is not declared as accessible resource and that it is broken. (I'm working on it)

**Pull request merged**